src/components/ClusterNodes/NodeDetailsList.tsx

```diff
@@ -486,54 +468,53 @@ export default function NodeDetailsList({ isSuperAdmin, renderRefreshBar, addTab
         }
     }
 
+    const gridTemplateColumns = `260px 180px repeat(${appliedColumns.length - 2}, 120px)`
+
     const renderNodeList = (nodeData: Object): JSX.Element => {
```

Comment:
In list in case of only one column not taking full width, and on hover there is layout shift

Reply:
fixed. The layout shift was due to copy button popping in on hover. p-4 has been added to copy button. But the kebabs (popup menu button) of the node list was missing the p-4.